src/core.js

```diff
@@ -154,8 +154,8 @@ define([
                     return; // abort
                 }
                 _this.ensureCurrentMugIsSaved(function () {
-                    if (!_.isUndefined(window.analytics)) {
-                        window.analytics.track("Clicked Save in Formbuilder");
+                    if (!_.isUndefined(window._kmq)) {
+                        window._kmq.push(["record", "Clicked Save in Formbuilder"]);
```

Comment:
Can we make a generic API in HQ to register analytics events? Ideally it would be the same API that *segment?* has (`window.analytics.track(EVENT_NAME)`) so we wouldn't have to change things in the form builder every time we decide to switch analytics engines. Also note that [copy-paste-dev](https://github.com/dimagi/Vellum/compare/copy-paste-dev?expand=0) (currently in QA) makes [some](https://github.com/dimagi/Vellum/compare/copy-paste-dev?expand=0#diff-815d962c2df9702e02af486ea3984d4bR188) [calls](https://github.com/dimagi/Vellum/compare/copy-paste-dev?expand=0#diff-815d962c2df9702e02af486ea3984d4bR201) [to](https://github.com/dimagi/Vellum/compare/copy-paste-dev?expand=0#diff-815d962c2df9702e02af486ea3984d4bR248) `window.analytics.track()` as well.